qutebrowser/browser/webview.py

```diff
@@ -145,13 +146,29 @@ def __init__(self, win_id, parent=None):
         self.loadProgress.connect(lambda p: setattr(self, 'progress', p))
         objreg.get('config').changed.connect(self.on_config_changed)
 
+    @pyqtSlot()
+    def on_initial_layout_complete(self):
+        """Add url to history now that we have displayed something."""
+        history = objreg.get('web-history')
+        # QUrl.None or .None_ in later versions is from the enum
+        # QUrl.UrlFormattingOption.
+        if not self._orig_url.matches(self.cur_url,
+                                      getattr(QUrl, 'None',
+                                              getattr(QUrl, 'None_'))):
```

Comment:
This is a bit unfortunate indeed... But `QUrl.UrlFormattingOption(0)` seems to work as well and seems cleaner.